lib/MusicBrainz/Server/Data/Recording.pm

```diff
@@ -301,7 +301,8 @@ sub find_standalone
 {
     my ($self, $artist_id, $limit, $offset) = @_;
     my $query ='
-        SELECT ' . $self->_columns . '
+        SELECT DISTINCT ' . $self->_columns . ',
+            musicbrainz_collate(recording.name)
```

Comment:
Distinct makes sense, but why add this?

Reply:
Below comment https://github.com/metabrainz/musicbrainz-server/pull/1443#discussion_r399800203 seems to answer that too.